### PR TITLE
Note on where to find Travis CI scripts for Julia

### PR DESCRIPTION
**Pull request by milesfrain**

### CODE REVIEW

**Comment by milesfrain** [comment]
Is @travis-ci/julia-maintainers the correct group to ping for "reviewers with write access"?

**Comment by Pezi777** [comment]
@logankilpatrick: Could you please have a look at this PR? thank you :-)

**Comment by logankilpatrick** [comment]
@milesfrain thanks for doing this. I’m not sure I follow your question above though. Can you elaborate on it?

**Comment by milesfrain** [comment]
@logankilpatrick My earlier question was referring to the merge message about the need for approval by "reviewers with write access".

image

**Comment by logankilpatrick** [comment]
Ah! That makes sense. @Pezi777 is that person :) since it’s the Travis docs repo, a Travis employee (of which @Pezi777 is one I believe) must approve in and merge the PR.

File: user/languages/julia.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Note that the `coverage=true` argument only tells `Pkg.test` to emit coverage in
 about the tests it ran; it does not submit this information to any services.
 To submit coverage information, see the coverage section above.
 
+The Travis CI scripts describing this default behaivor are [julia.rb](https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb) and [julia_spec.rb](https://github.com/travis-ci/travis-build/blob/master/spec/build/script/julia_spec.rb).
```
Comment (Pezi777) [inline user/languages/julia.md]:
```suggestion
The Travis CI scripts describing this default behavior are [julia.rb](https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/julia.rb) and [julia_spec.rb](https://github.com/travis-ci/travis-build/blob/master/spec/build/script/julia_spec.rb).
```

**Comment by milesfrain** [comment]
Force pushed with typo fix squashed with original to reduce commit history spam.

**Comment by milesfrain** [comment]
@logankilpatrick I'm happy to change the wording if you have another suggestion.

**Comment by Pezi777** [comment]
thank you so much @milesfrain

**Pull request merged**